pennylane/interfaces/autograd.py

```diff
@@ -111,15 +111,12 @@ def _execute(
 
     for i, r in enumerate(res):
 
+        if any(m.return_type is qml.measurements.Counts for m in tapes[i].measurements):

```

Comment:
Every interface was updated to skip the conversion to a tensor object if Counts was one of the measurements.